cmd/clef/main.go

```diff
@@ -459,6 +489,43 @@ func initialize(c *cli.Context) error {
 	return nil
 }
 
+func listAccounts(c *cli.Context) error {
+	internalApi, err := initInternalApi(c)
+	accs, err := internalApi.ListAccounts(context.Background())
+	if err != nil {
+		return err
+	}
+	if len(accs) == 0 {
+		fmt.Println("\nThe keystore is empty.")
+	}
+	fmt.Println()
+	for _, account := range accs {
+		fmt.Println(account.Address)
```

Comment:
If you feel strongly about it, we can leave it like this, otherwise I'd rather print the account.URL too

Reply:
It's odd -- this was somehow reverted back to an earlier version

Reply:
Ah, you reverted it here: https://github.com/ethereum/go-ethereum/pull/26080/commits/b8e04dd7616390b549b540e7359d5c8d30313920#diff-c08df6df90e891223dbfc3d85873d44064604480bb9b72a516facab85472aa6eR408 
Intentional?

Reply:
ah, no, not intentional. 
```suggestion
		fmt.Printf("%v (%v)\n", account.Address, account.URL)
```